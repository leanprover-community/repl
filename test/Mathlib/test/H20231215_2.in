{"unpickleEnvFrom": "test/H20231215.olean"}

{"cmd": "theorem mathd_numbertheory_739 :\n  9! % 10 = 0 := by sorry", "env": 0, "sorries": "individual"}
