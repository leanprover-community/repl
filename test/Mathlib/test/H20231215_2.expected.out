{"env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 2, "column": 20},
   "parentDecl": "mathd_numbertheory_739",
   "goals": ["‚ä¢ 9! % 10 = 0"],
   "endPos": {"line": 2, "column": 25}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 30},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

